app/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/LandingPageController.scala

```diff
@@ -42,10 +41,8 @@ class LandingPageController @Inject() (
     Action(implicit request => Ok(landing_page(indirectDisposalsEnabled, mixedUseEnabled)))
 
   def agentsLandingPage(): Action[AnyContent] =
-    Action(implicit request =>
-      Ok(
-        agents_landing_page(indirectDisposalsEnabled, mixedUseEnabled)
-      )
+    Action(
+      Redirect(viewConfig.agentsStartPageUrl).withNewSession
```

Comment:
we might not necessarily need `.withNewSession` here - is there a particular reason you put it in?

Reply:
No, somewhere in landing page logic I saw it. I thought its required here